src/vcpkg/export.ifw.cpp

```diff
@@ -424,19 +420,17 @@ namespace vcpkg::Export::IFW
         const auto ifw_packages_dir_path = get_packages_dir_path(export_id, ifw_options, paths);
 
         fs.remove_all(ifw_packages_dir_path, ec, failure_point);
-        Checks::check_exit(VCPKG_LINE_INFO,
-                           !ec,
-                           "Could not remove outdated packages directory %s due to file %s",
-                           ifw_packages_dir_path,
-                           failure_point);
+        Checks::msg_check_exit(
+            VCPKG_LINE_INFO,
+            !ec,
+            msg::format(msgRemovingPackageDirFailed, msg::path = ifw_packages_dir_path).append_raw(failure_point));
```

Comment:
I think this should do:

```
fs.remove_all(ifw_packages_dir_path, VCPKG_LINE_INFO);
```

and delete `RemovingPackageDirFailed`.

Reply:
So rather than using the check exit, 
```
if (ec) 
{
    fs.remove_all(ifw_packages_dir_path, VCPKG_LINE_INFO);
}
```
?

Reply:
No, just:

```
fs.remove_all(ifw_packages_dir_path, VCPKG_LINE_INFO);
```

There's no need for anything related to EC when using the LineInfo overloads of the filesystem things; they already terminate with a message on failure.